client/component-lab/SubscriptionNoCardWarning/SubscriptionNoCardWarning.coffee

```diff
@@ -0,0 +1,29 @@
+kd = require 'kd'
+React = require 'react'
+
+Box = require 'lab/Box'
+Label = require 'lab/Text/Label'
+{ Grid, Row, Col } = require 'react-flexbox-grid'
```

Comment:
Yep, I am using it elsewhere, but it looks like it's an overkill here. I will send a fix.